core/src/main/java/com/linecorp/armeria/client/retry/AbstractRetryingClient.java

```diff
@@ -52,11 +51,11 @@
 /**
  * A {@link Client} decorator that handles failures of remote invocation and retries requests.
  *
- * @param <I> the {@link Request} type
- * @param <O> the {@link Response} type
+ * @param <REQ_T> the {@link Request} type
+ * @param <RES_T> the {@link Response} type
  */
-public abstract class AbstractRetryingClient<I extends Request, O extends Response>
-        extends SimpleDecoratingClient<I, O> {
+public abstract class AbstractRetryingClient<REQ_T extends Request, RES_T extends Response>
+        extends SimpleDecoratingClient<REQ_T, RES_T> {
```

Comment:
Could you revert these changes? 😅 We conventionally use `I` and `O` in other classes.

Reply:
Done